webapp-django/crashstats/crashstats/static/crashstats/js/socorro/correlation.js

```diff
@@ -169,7 +180,17 @@ window.correlations = (function () {
             .map(function (line) {
                 var percentGroup = toPercentage(line.count_group / total_group);
                 var percentRef = toPercentage(line.count_reference / total_reference);
-                return '(' + percentGroup + '% in signature vs ' + percentRef + '% overall) ' + itemToLabel(line.item);
+
+                var result = '(' + percentGroup + '% in signature vs ' + percentRef + '% overall) ' + itemToLabel(line.item);
+
+                // If the rule has a prior that alters its distribution significantly, print it after the rule.
+                if (line.prior) {
+                    var percentGroupGivenPrior = toPercentage(line.prior.count_group / line.prior.total_group);
+                    var percentRefGivenPrior = toPercentage(line.prior.count_reference / line.prior.total_reference);
```

Comment:
You might want to make sure that ``line.prior.total_group`` and ``line.prior.total_reference`` are different than zero, to prevent possible mistakes.